src/detail_screen/anime.css

```
@@ -0,0 +1,267 @@
+@import url('https://fonts.googleapis.com/css2?family=Quicksand:wght@300;400&family=Roboto:wght@100;300;400&display=swap');
+
+:root {
+  --grey-1: #A9A9BA;
+  --grey-2: #6F6F7A;
+  --grey-3: #35353B;
+  --grey-4: #2E2E33;
+  --grey-5: #2A2A2E;
+  --purple: #9572F8;
+}
+
+body {
+  margin: 0px;
```

Só pra deixar mais limpo
```suggestion
  margin: 0;
```